src/bin/heroku.rs

```diff
@@ -31,9 +35,39 @@ use std::{env, net::IpAddr};
 #[tokio::main]
 async fn main() -> Result<(), Box<dyn std::error::Error + Send + Sync>> {
 	env_logger::init();
+
+	dotenv().expect(".env file with PGPASSWORD variable");
+	let pg_password = env::var("PGPASSWORD").unwrap();
+	let connection_uri = format!("postgres://postgres:{}@localhost/postgres", pg_password);
```

Comment:
Fixed this and updated instructions to use `DATABASE_URL` because that allows sqlx to perform compile time verification of queries. https://github.com/launchbadge/sqlx#compile-time-verification